.gitignore

```diff
@@ -83,3 +83,5 @@ hs_err_pid*
 .mtj.tmp/
 # blueJ files
 *.ctxt
+
+*swagger-ui*
```

Comment:
seems no need to add here due to the master code has fixed it (#2277)

Reply:
and also need handle the CI error
![image](https://github.com/apache/incubator-hugegraph/assets/17706099/f4c84ffd-5343-47ee-a90b-d36351b1620e)